pallets/inflation-rewards/src/mock.rs

```diff
@@ -161,18 +163,29 @@ impl Default for Mocks {
 
 pub struct MockContainerChainGetter;
 
-impl tp_traits::GetCurrentContainerChains for MockContainerChainGetter {
-    type MaxContainerChains = ConstU32<5>;
+impl tp_traits::GetContainerChainsWithCollators<AccountId> for MockContainerChainGetter {
```

Comment:
As I said it might make more sense to put the maxLengthParaIds inside a type `GetContainerChainsWithCollators`, like `GetContainerChainsWithCollators` had